core-libs/setup/ssr/optimized-engine/optimized-ssr-engine.ts

```diff
@@ -59,7 +59,13 @@ export class OptimizedSsrEngine {
   constructor(
     protected expressEngine: NgExpressEngineInstance,
     protected ssrOptions?: SsrOptimizationOptions
-  ) {}
+  ) {
+    this.log(
```

Comment:
I guess, we want to pass `true` as a second argument?
Otherwise the options are not logged, unless you configure `debug:true` option.

Reply:
I believe it should be logged only when `debug` is set to `true` :)